tests/Doctrine/Tests/DBAL/Functional/Driver/PDOSqlite/DriverTest.php

```diff
@@ -25,6 +25,14 @@ protected function setUp() : void
         $this->markTestSkipped('pdo_sqlite only test.');
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function testReturnsDatabaseNameWithoutDatabaseNameParameter()
+    {
+        $this->markTestSkipped('SQLite does not support the concept of a database.');
```

Comment:
Previously, this test would only pass with an in-memory SQLite database for which the `Driver::getDatabase()` would return NULL. The test is invalid with SQLite by design.